optimum/onnxruntime/modeling_ort.py

```diff
@@ -138,6 +143,12 @@ def load_model(path: Union[str, Path], provider=None):
                 ONNX Runtime provider to use for loading the model. Defaults to `CPUExecutionProvider`.
         """
         if provider is None:
+            if check_if_multiple_available_providers():
+                raise ValueError(
+                    "No provider supplied and multiple providers available, "
+                    "can not deterministically set correct provider. "
+                    "Pass provider in explicitly."
+                )
```

Comment:
@philschmid I added this check here so that in the future if you have multiple providers available and don't supply a provider like I did, `load_model` won't fail for nebulous ORT reasons

Reply:
Not sure about that. Since when you install `onnxruntime-gpu` you always have 2 providers. We should keep the current behavior that if you run `model.to` the model is moved to GPU (similar to `transformers`) and if you don'T define anything it is CPU.